numpyro/distributions/continuous.py

```diff
@@ -1680,6 +1680,70 @@ def infer_shapes(df, loc, scale_tril):
         return batch_shape, event_shape
 
 
+class GaussianCopula(Distribution):
+    arg_constraints = {
+        "covariance_matrix": constraints.positive_definite,
+        "precision_matrix": constraints.positive_definite,
+        "scale_tril": constraints.lower_cholesky,
+    }
+    support = constraints.real_vector
+    reparametrized_params = [
+        "covariance_matrix",
+        "precision_matrix",
+        "scale_tril",
+    ]
+
+    support = constraints.unit_interval
+
+    def __init__(
+        self,
+        transform_dist,
```

Comment:
Looking at your implementation, I guess we can call this `marginal_dist`?

Reply:
Good idea, I'll rename it to that.